dev/github/run_tests.sh

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env bash
+#
+# The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+# (the "License"). You may not use this work except in compliance with the License, which is
+# available at www.apache.org/licenses/LICENSE-2.0
+#
+# This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied, as more fully set forth in the License.
+#
+# See the NOTICE file distributed with this work for information regarding copyright ownership.
+#
+
+#
+# This script is run from inside the Docker container
+#
+set -ex
+
+if [ -n "${ALLUXIO_GIT_CLEAN}" ]
+then
+  git clean -fdx
+fi
+
+mvn_args=""
+if [ -n "${ALLUXIO_MVN_RUNTOEND}" ]
+then
+  mvn_args+=" -fn -DfailIfNoTests=false --fail-at-end"
+fi
+
+RUN_MAVEN="false"
```

Comment:
this is a copy of the bash code in `run_checks.sh`; can they be consolidated?

Reply:
I don't intend to consolidate them. It's very very annoying to consolidate shell code in a reasonable way

Reply:
though I can probably delete a lot of things here so gonna do that first.